lib/network.ts

```diff
@@ -20,25 +21,33 @@ export type UrlResult = {
 
 export const getUrlStatus = async (
   url: string,
-  timeoutSeconds = WEB_REQUEST_TIMEOUT_SECONDS
-) => {
+  timeoutSeconds = WEB_REQUEST_TIMEOUT_SECONDS,
+  proxyUrl?: string,
```

Comment:
think we should move `timeoutSeconds` to be the last param, because usually it will not be used